set_bucket_tags/app.py

```diff
@@ -106,37 +106,16 @@ def create_or_update(event, context):
     )
   log.debug(f'Tagging response: {tagging_response}')
   if tagging_response.get('ResponseMetadata').get('HTTPStatusCode') == 204:
-    log.debug('got 204 response, all is fine')
+    log.debug('Got 204 response, all is well')
     return True
   else:
     log.debug('Tagging failed')
-    status = 'failure'
-    reason = 'tagging failed'
-    data = tagging_response
-    send_response(event, context, status, reason, data)
     return False
```

Comment:
my experience is that returning a `False` to cloudformation is not the same thing as a FAILURE.  It will just treat it as a SUCCESS.  To let CFN know that it should fail and rollback you need to explicitly send a failure response using the send_repsone method

Reply:
@zaro0508 Looking at your actions CR, you don't do this either: https://github.com/Sage-Bionetworks/aws-infra/blob/master/lambdas/cfn-sc-actions-provider/sc-actions-provider/app.py#L88

Given that is the case, what do you advise for best practice both when there's something arguably meaningful to return, as is the case for your CR, and when there isn't, as is the case for mine?

Reply:
@zaro0508 I don't need to raise an exception here because I can rely on the client to do that for me -- so I removed even more code. See 7a60c5d